addons/web/static/src/views/relational_model/relational_model.js

```diff
@@ -415,31 +422,30 @@ export class RelationalModel extends Model {
             delete group.__count;
             delete group[`${firstGroupByName}_count`];
             delete group.__range;
-            group.value = group[config.groupBy[0]];
+            group.value = getValueFromGroupData(group, groupByField, group[config.groupBy[0]]);
```

Comment:
could the third argument be deducted from the first two? If so, please remove it.